util/check-format-test-negatives.c

```diff
@@ -22,9 +23,19 @@
         return;                                 \
     }
 
+/* allow extra  SPC in single-line comment */
 /*-
- * allow extra SPC in format-tagged multi-line comment
+ * allow extra  SPC in regular multi-line comment
```

Comment:
Isn't this comment still format-tagged?

Reply:
Thanks for your comments.
Here I had made a copy/paste error - fixed.